zk_toolbox/crates/zk_inception/src/commands/chain/args/init.rs

```diff
@@ -28,6 +28,8 @@ pub struct InitArgs {
     pub deploy_paymaster: Option<bool>,
     #[clap(long, help = MSG_L1_RPC_URL_HELP)]
     pub l1_rpc_url: Option<String>,
+    #[clap(long, help = MSG_PORT_OFFSET_HELP)]
+    pub port_offset: Option<u16>,
```

Comment:
let's apply offset based on chain_config.id * 100

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2594/commits/ef360c39b5239636326ab9f65aaad17c66568804

Reply:
Improved in https://github.com/matter-labs/zksync-era/pull/2594/commits/32da4b84bee2d3f8f08cf257edaea53c4dbfdceb